modules/fluid_properties/unit/src/LeadBismuthFluidPropertiesTest.C

```diff
@@ -0,0 +1,128 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "LeadBismuthFluidPropertiesTest.h"
+#include "SinglePhaseFluidPropertiesTestUtils.h"
+
+/**
+ * Test that the fluid name is correctly returned
+ */
+TEST_F(LeadBismuthFluidPropertiesTest, fluidName) { EXPECT_EQ(_fp->fluidName(), "LeadBismuth"); }
+
+/**
+ * Test that the molar mass is correctly returned
+ */
+TEST_F(LeadBismuthFluidPropertiesTest, molarMass)
+{
+  ABS_TEST(_fp->molarMass(), 233.99, REL_TOL_SAVED_VALUE);
+}
+
+/**
+ * Verify calculation of the LeadBismuth fluid properties
+ */
+TEST_F(LeadBismuthFluidPropertiesTest, properties)
+{
+  const Real tol = REL_TOL_EXTERNAL_VALUE;
+  const std::vector<Real> pressures = {1e5, 1e6, 5e6};
+  const std::vector<Real> temperatures = {400, 500, 700};
+
+  // Solutions : obtained from running the property object itself
+  // Verified visually against the reference
```

Comment:
I'm confused. You got these values from the object you're testing? I know you haven't used full precision in these values, but your tolerances seem pretty loose. `REL_TOL_EXTERNAL_VALUE` is 1e-3. It's not meant for comparing to values from your own code. That's what `REL_TOL_SAVED_VALUE` (1e-12) is recommended for (requires outputting your values in all or most of their precision).